frontend/src/app/page.tsx

```
@@ -216,11 +216,17 @@ export default function Page() {
       ) : (
         <div className='content-list'>
           {personalizedContent.map((content, index) => (
-            <ContentTile
-              key={content.uid || index}
-              content={content}
-              index={index}
-            />
+            <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)